atest/robot/output/xunit.robot

```diff
@@ -71,6 +84,43 @@ Skipping non-critical tests is deprecated
     Run tests    --xUnit xunit.xml --xUnitSkipNonCritical     ${PASS AND FAIL}
     Stderr Should Contain   Command line option --xunitskipnoncritical has been deprecated and has no effect.
 
+Nullish timestamp prerebotmodifier
+    Nullish Timestamp Should Be    ${EMPTY}
+    
+Invalidated suite status
+    Run Tests Without Processing Output    -l log.html    ${PASS AND FAIL}
+    Remove Suite Status Attribute    ${OUTFILE}    name=starttime
+    Execute On Existing Execution Environment
+    ...    ${INTERPRETER.rebot}    --xUnit xunit.xml -l log.html    ${OUTFILE}    ${COMMON DEFAULTS}
+    Verify Xfile Timestamp    ${EMPTY}
+
+Missing suite status
+    Run Tests Without Processing Output    -l log.html    ${PASS AND FAIL}
+    Remove Suite Status Element    ${OUTFILE}
+    Execute On Existing Execution Environment
+    ...    ${INTERPRETER.rebot}    --xUnit xunit.xml -l log.html    ${OUTFILE}    ${COMMON DEFAULTS}
+    Verify Xfile Timestamp    ${EMPTY}
+
+Merge outputs
+    Run Tests Without Processing Output    --output ${MERGE ONE}    ${PASS AND FAIL}
+    Run Tests Without Processing Output    --output ${MERGE TWO}    ${TESTDATA}
+    Execute On Existing Execution Environment
+    ...    ${INTERPRETER.rebot}    --xUnit xunit.xml -l log.html    ${MERGE TWO} ${MERGE ONE}    ${COMMON DEFAULTS}
+    Verify Xfile Timestamp    ${EMPTY}
+
+Merge outputs with times
+    Run Tests Without Processing Output    --output ${MERGE ONE}    ${PASS AND FAIL}
+    Run Tests Without Processing Output    --output ${MERGE TWO}    ${TESTDATA}
+    Execute On Existing Execution Environment
+    ...    ${INTERPRETER.rebot}    ${OPTIONS WITH TIMES}    ${MERGE TWO} ${MERGE ONE}    ${COMMON DEFAULTS}
+    Verify Xfile Timestamp    ${XTIMESTAMP}
+
+Rebot with start and end time
+    Run Tests Without Processing Output    -l log.html    ${PASS AND FAIL}
+    Execute On Existing Execution Environment
+    ...    ${INTERPRETER.rebot}    ${OPTIONS WITH TIMES}    ${OUTFILE}    ${COMMON DEFAULTS}
+    Verify Xfile Timestamp    ${XTIMESTAMP}
```

Comment:
These tests look fine but I don't understand the need to use the new `Execute On Existing Execution Environment` keyword. As far as I can the normal `Run Rebot` would work.

With the last test it probably would be possible to use also `Run Tests`, but then we needed to create the expected xunit timestamp based on suite start time ourselves. Something like this could possibly work:

```
Run Tests    --xunit xunit.xml    ${PASS AND FAIL}
Verify XUnit Timestamp    ${{datetime.datetime.strptime($SUITE.starttime, '%Y%m%d %H:%M:%S.%f').strftime('%Y-%m-%dT%H:%M:%S.%f')}}
```
Yeah, that timestamp conversion is quite complicated. With tests using Rebot specifying the start time using ´--starttime´ is a very good idea.

Reply:
I feel ```Merge outputs with times``` and ```Rebot with start and end time``` are falling to very same test in the end. Removing the merge one.